CONTRIBUTING.md

```diff
@@ -1,39 +1,38 @@
 # Introduction
 
-**Thank you** for considering to contribute to CFF!
+**Thank you** for considering to contribute to the **Citation File Format (CFF) and its schema**!  
+If you mean to contribute to another part of the Citation File Format project, 
+for example a specific software package for working with CFF files,
+please contribute to the respective repository ([list of repositories in the `citation-file-format` GitHub organization](https://github.com/orgs/citation-file-format/repositories)).
 
 **Please follow these guidelines.** Their purpose is to make both contributing and accepting contributions easier for all parties involved.
 
 There are many **ways to contribute**, e.g.:
 
-- Write blog posts, tutorials, etc. about CFF
-- Review any and all parts of the format and its infrastructure: specifications, libraries, documentation, etc., and submit bug reports, feature requests, suggestions and similar, respectively
+- Tell a friend or colleague about the Citation File Format, or tweet about it
+- Write blog posts, tutorials, etc. about the Citation File Format
+- Review the format and its schema dn documentation
 - Language polish any prose output, including the specifications
```

Comment:
I'd say "Improve wording of any prose output, including the specifications"

Reply:
Done in f251ae8.